src/OnboardingSPA/components/StartOptions/index.js

```diff
@@ -60,6 +68,29 @@ const StartOptions = ( { questionnaire, oldFlow, options } ) => {
 		}
 		navigate( data.steps[ 1 ].path );
 	};
+
+	const handleMigration = () => {
+		if ( canMigrateSite ) {
+			const migrationStepExists = allSteps.some(
+				( step ) => step.path === stepSiteGenMigration.path
+			);
+
+			if ( ! migrationStepExists ) {
+				const updates = injectMigrationStep( allSteps, stepTheFork );
+				updateAllSteps( updates.allSteps );
+			}
+			trackOnboardingEvent(
+				new OnboardingEvent(
+					ACTION_SITEGEN_FORK_OPTION_SELECTED,
+					'MIGRATE'
+				)
+			);
```

Comment:
@arunshenoy99 Can you explain the difference between this event and the one below that uses the same event type, but has a label value of `MIGRATION` vs `MIGRATE`?

Reply:
Nevermind, I see this was moved from a different location. This part is fine, the other part of the code shouldn't be there because we are now double counting these events and sending a newer label value of `MIGRATION` that was not being tracked before.

Reply:
https://github.com/newfold-labs/wp-module-onboarding/pull/606